api/src/api/incidents.js

```diff
@@ -0,0 +1,34 @@
+const router = require('express').Router();
+const Incident = require('../models/Incident');
+
+router.get("/city/:city", async (req, res) => {
```

Comment:
Instead of creating an endpoint for each query param, let's only define one endpoint and use `req.query` to grab the query params (see [this](https://masteringjs.io/tutorials/express/req) for more info).

Reply:
@kenetec - Updated, so now we only have one endpoint for all query params!